src/includes/class-healthcheck.php

```diff
@@ -63,6 +63,16 @@ public function init() {
 		add_action( 'wp_ajax_health-check-files-integrity-check', array( 'Health_Check_Files_Integrity', 'run_files_integrity_check' ) );
 		add_action( 'wp_ajax_health-check-view-file-diff', array( 'Health_Check_Files_Integrity', 'view_file_diff' ) );
 		add_action( 'wp_ajax_health-check-mail-check', array( 'Health_Check_Mail_Check', 'run_mail_check' ) );
+		add_action( 'wp_ajax_health-check-wp-debug-create-backup', array( 'Health_Check_WP_Debug', 'create_wp_config_backup' ) );
```

Comment:
There's way too many ajax actions, the user shouldn't need to worry about most of these things, ideally they need a enable, disable and truncate button, and we need a 4th action that grabs log content live if desired.